unfoldingWord/tc-create-app
# Supporting version 1.5

Testing Notes:
1. verify that Cut-paste works 
2. In a TN file, verify that support reference is a drop down that you can select any value that currently exists in the file (ie, should match the list in the search filter)
3. Verify that all filterable values have drop downs too (but I caution about actually changing them since the row would no longer be in the expected location if you changed chapter or verse in TN or reference in TWL)
4. Type a new value instead of selecting (I use "figbar")
5. Then click to add a row
6. Verify that "figbar" is in the list
7. Save and return to the file to confirm changes persisted.
